CMakeLists.txt

```diff
@@ -447,11 +447,12 @@ else()
 endif()
 
 include(FindPkgConfig)
-pkg_check_modules(GSTREAMER IMPORTED_TARGET gstreamer-sdp-1.0>=1.16 gstreamer-webrtc-1.0>=1.16)
+pkg_check_modules(GSTREAMER IMPORTED_TARGET gstreamer-sdp-1.0>=1.18 gstreamer-webrtc-1.0>=1.18)
 if (TARGET PkgConfig::GSTREAMER)
+	pkg_check_modules(XCB IMPORTED_TARGET xcb xcb-ewmh)
```

Comment:
Can you output a configure message for this too? So that people can see, if this is enabled?

Reply:
Will do.